Turf/Turf.swift

```diff
@@ -285,3 +285,65 @@ public struct Polyline {
         return closestCoordinate
     }
 }
+
+public struct Polygon {
+    var radius:Float = 6378137
+    var coordinates: [CLLocationCoordinate2D]
+    
+    var polygonArea: Float {
```

Comment:
Let's change all Float to Double so we don't have to cast when converting to radians.

Reply:
After a second thought, this should probably be `CLLocationDistance`